### PR TITLE
store userName in localStorage

### LINKED ISSUE #26: Code for User login & Store User Name
Relates #8 

**objective:** Store userName in local storage.

**_in welcomePage.js file:_**
- [x] Declare variables (inputName & btnLogin) using DOM to access tags.
- [x] Create a function that works when clicking on btnLogin.
     - **_This function contains:_**
          - condition to check input tag if empty or not.
               - if the input is empty, then you should display an alert 'Enter Your Name Please' for the user.
               - else, should store userName in local Storage and convert a user to mainPage.

### PR DESCRIPTION
**Pull request by EmanHass**
relates #26

### CODE REVIEW

**Review by abdulrahman-2020 — DISMISSED** [review]
great!

File: WelcomePage/WelcomePage.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const inpName = document.getElementsByClassName('inpName')[0];
+const btnLogin = document.getElementsByClassName('btnLogin')[0];
+
+btnLogin.onclik = welcome;
+
+function welcome(e){
+    if(inpName.value == '') 
+        alert('Enter Your Name Please');
```
Comment (MMansour87) [inline WelcomePage/WelcomePage.js:8]: I think internal {} needed for the if

File: WelcomePage/WelcomePage.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const inpName = document.getElementsByClassName('inpName')[0];
+const btnLogin = document.getElementsByClassName('btnLogin')[0];
+
+btnLogin.onclik = welcome;
```
Comment (MMansour87) [inline WelcomePage/WelcomePage.js]: I think it is onclik not onclick
I am not sure if welcome needs () or not

**Review by MMansour87 — CHANGES_REQUESTED** [review]
Thank you for your efforts
Please read my humble comments

File: WelcomePage/WelcomePage.js (hunk 1 above)
Comment (EmanHass) [reply WelcomePage/WelcomePage.js:8]: when if condition take one statement, we don't need to put {}

File: WelcomePage/WelcomePage.js (hunk 2 above)
Comment (EmanHass) [reply WelcomePage/WelcomePage.js]: Oh, yes it's onclick not onclik, sorry
I will rewrite it.
No, we don't need to put () when
 invoke function

Comment (abdulrahman-2020) [reply WelcomePage/WelcomePage.js]: Line 4 I expect we have to put () in order to call the function

**Review by mo7amedehab97 — APPROVED** [review]

**Review by abdulrahman-2020 — DISMISSED** [review]

**Review by lina-jamal — APPROVED** [review]

**Review by abdulrahman-2020 — APPROVED** [review]

File: WelcomePage/WelcomePage.js (hunk 1 above)
Comment (MMansour87) [reply WelcomePage/WelcomePage.js:8]: New Info, thank You

**Pull request merged**